meta-mender-qemu/recipes-bsp/u-boot/u-boot-vexpress-qemu.inc

```diff
@@ -1,5 +1,17 @@
 FILESEXTRAPATHS_prepend := "${THISDIR}/patches:${THISDIR}/files:"
 
+# Remove this block when moving from dunfell to the next branch, and also remove
+# the bundled version of u-boot_2021.07. See MEN-3915.
```

Comment:
Please make a note for this in the task itself too.

Reply:
I will. I just want to get this committed first so I can refer to the commit IDs.

Reply:
On second thought, I will change this comment. The revert of the bundled version is now tracked implicitly by being included as a revert in [the master-next branch](https://github.com/mendersoftware/meta-mender/commit/cd81dfa80e1d6dbb80a0d02c8b7af7b5471af85a). The removal of the patches listed here doesn't actually depend on which branch we're on, but on whether upstream accepts our patches. I will create a task for tracking the upstream work, and then link to that from here.

Reply:
https://github.com/mendersoftware/meta-mender/pull/1460 and https://tracker.mender.io/browse/MEN-5023